### PR TITLE
Module2 homework

### PR DESCRIPTION
**Pull request by markrenaud**
Completed homework Parts 1, 2 & 3.

### CODE REVIEW

File: Module-2/README.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+-  **Reduced maintainability** (if the same number is used in multiple places, it needs to be changed everywhere it occurs; are all cases where that number is used representing the same concept?).
+- **Difficulty debugging** (if the magic number is causing the issue, where did it come from, and why is it there?).
+
+Magic numbers can be avoided by using named constants or creating sets of related constants (e.g., using enums or static variables in structs).
+
+#### _h) How can you view your App in Light and Dark Modes simultaneously?_
+
+- Choosing the `Color Scheme Variants` in the Xcode Canvas.
+- Creating multiple previews in the `PreviewProvider` for the `View` with associated `.preferredColorScheme(_ colorScheme: ColorScheme?) ` modifiers and switch between the two.
+
```
Comment (niknovak) [inline Module-2/README.md:71]: Just a note here if someone picky asks ;)
In Swift by convention, the documentation is always referring to methods by omitting parameter names like `colorScheme` and types like `ColorScheme`. So we would write it like this `preferredColorScheme(_:)`.

File: Module-2/README.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+#### _i) Below is an image of the Canvas from Xcode. The Canvas is in selectable mode. Can you explain why the red background does not cover the entire button area?_
+
+The order modifiers applied to a View matters.
+
+The shown result is likely due to a `padding` modifier being applied **after** a `background` modifier.
+
+By moving the `background` modifier **before** the `padding` modifier, the desired outcome can be achieved.
+
```
Comment (niknovak) [inline Module-2/README.md:79]: You mean after, right? ;)

**Review by niknovak — APPROVED** [review]
👌

File: Module-2/README.md (hunk 1 above)
Comment (markrenaud) [reply Module-2/README.md:71]: Noted.  Will do in the future.  Thanks.

File: Module-2/README.md (hunk 2 above)
Comment (markrenaud) [reply Module-2/README.md:79]: Yes.  It should have read:  
```
By moving the `padding` modifier **before** the `background` modifier, the desired outcome can be achieved.
```

**Pull request merged**